src/main/java/seedu/address/logic/commands/addcommand/AddEventCommand.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands.addcommand;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GENERAL_EVENT;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.GeneralEvent;
+
+public class AddEventCommand extends AddCommand {
```

Comment:
If this is only for GeneralEvent, is it better to have AddGeneralEventCommand instead to be more specific and clear